### ISSUE TITLE
How to i send bitcoin even though i have 0 unspent transactions

### ISSUE THREAD

**Issue by Chriszat** [ofek/bit#143]

**Comment by Alex-CodeLab** [comment]
why?

**Comment by Alex-CodeLab** [comment]
impossible.
I you want to try how broadcasting works, then use Testnet, and request some free testnet coins.

**Comment by Chriszat** [comment]
I saw an application where you could send bitcoin to an address, but the
transaction never gets confirmation on the block chain. So I am trying to
recreate the application using python. So I was wondering how one can send
a broadcast a transaction to the blockchain that will never get a
confirmation.

Thanks.

**Comment by teran-mckinney** [comment]
Most invalid transactions will get rejected at the pushtx APIs.

You might have to run a Bitcoin daemon to try pushing something invalid, and it'll just get dropped.

You could try to use previously spent inputs and push that, maybe.

**Comment by Chriszat** [comment]
Thank you for you help, but I was wondering if you can help me with a
python code example on how to do this, or you can show me some apis to use.
I would really appreciate it. Am just looking to learn more.


Thank you for your time.

**Comment by Alex-CodeLab** [comment]
> So I am trying to recreate the application using python. So I was wondering how one can send a broadcast a transaction to the blockchain that will never get a confirmation.

No, this is not possible. It will be rejected if it is not a valid transaction. And if you have 0 bitcoin you can not make a valid transaction.
